src/app/redux/Login/reducers.js

```diff
@@ -1,29 +1,24 @@
+import { completeState, createReducer, completeReducer } from 'redux-recompose';
 import Immutable from 'seamless-immutable';
 
+import userService from '../../../services/UserService';
+
 import { actions } from './actions';
 
-const initialState = {
+const stateDescription = {
   isLoading: false
 };
 
-function reducer(state = Immutable(initialState), action) {
-  switch (action.type) {
-    case actions.USER_LOGIN_LOADING:
-      return state.merge({ isLoading: true });
-    case actions.USER_LOGIN_SUCCESS: {
-      return state.merge({
-        isLoading: false
-      });
-    }
-    case actions.USER_LOGIN_FAILURE: {
-      return state.merge({ isLoading: false });
-    }
-    case actions.USER_LOGOUT: {
+const initialState = completeState(stateDescription);
```

Comment:
you should add `isLoading` as the second param of this function, so it doesnt create `isLoadingLoading` and `isLoadingError` in the state.

`const initialState = completeState(stateDescription, ['isLoading']);`